package.json

```diff
@@ -27,19 +27,27 @@
   },
   "files": [
     "dist/*",
-    "src/*"
+    "src/*",
+    "registry/hardhat.config.ts",
+    "registry/package.json",
+    "registry/package-lock.json",
+    "registry/hardhat.config.js",
+    "registry/tsconfig.json",
+    "registry/contracts/*",
+    "registry/scripts/*"
   ],
   "scripts": {
-    "test": "node --experimental-fetch --experimental-vm-modules node_modules/.bin/jest --verbose",
-    "lint": "eslint src/ test/ --ext .js,.jsx,.ts,.tsx",
+    "postinstall": "mkdir -p registry && cd registry && npm install",
```

Comment:
The registry directory holds the functionality for running a hardhat node and deploying the registry if no path to an evm-tableland repo is provided.  It has it's own package.json and needs the `npm install` command to be run anytime `npm install` is run in the root directory.